src/wallet/rpcdump.cpp

```diff
@@ -601,6 +601,7 @@ UniValue dumpwallet(const JSONRPCRequest& request)
         throw std::runtime_error(
             "dumpwallet \"filename\"\n"
             "\nDumps all wallet keys in a human-readable format.\n"
```

Comment:
@TheBlueMatt would changing this string to something like `Dumps all wallet _derived_ keys in a human-readable format.` help cleanup the confusion?